maps-app-ios/UI/Account View/Account.storyboard

```diff
@@ -151,17 +144,17 @@
                         <viewControllerLayoutGuide type="bottom" id="ptj-BX-B1N"/>
                     </layoutGuides>
                     <view key="view" contentMode="scaleToFill" id="57D-QU-6n5">
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <rect key="frame" x="0.0" y="0.0" width="600" height="600"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <containerView hidden="YES" opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="xSs-iQ-f4q" userLabel="Logged In Container View">
-                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
+                                <rect key="frame" x="0.0" y="20" width="600" height="580"/>
                                 <connections>
                                     <segue destination="gYN-bY-eoQ" kind="embed" id="iGb-pJ-egr"/>
                                 </connections>
                             </containerView>
                             <containerView hidden="YES" opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Um7-OB-RNk" userLabel="Logged Out Container View">
-                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
+                                <rect key="frame" x="0.0" y="20" width="600" height="580"/>
```

Comment:
Do these dimension changes matter? Or are they just a function of the displayed device in the storyboard editor? I think it's fine, but they do distract from the meat of the change. Don't worry about it if it's fine and doesn't bother you.

Reply:
> Or are they just a function of the displayed device in the storyboard editor?

Correct, this is an Xcode automatically generated change. This doesn't excuse, however, how it made it into this PR. I will be more diligent in the future!

Reply:
It's quite possible that this is a function of my original commits of the app including device-specific stuff that they shouldn't have.